mautic/mautic
# Change btn-primary class of email buttons in campaign send email action to btn-default

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [No]
| New feature/enhancement? (use the a.x branch)      | [Yes]
| Deprecations?                          | [No]
| BC breaks? (use the c.x branch)        | [No]
| Automated tests included?              | [No] 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The 3 buttons on the send email action are using the class btn-default instead of btn-primary

The current view:
![image](https://user-images.githubusercontent.com/96085911/152776502-d54e7e17-aa89-40a6-9c05-68ec72e661d6.png)

The suggestion:
![image](https://user-images.githubusercontent.com/96085911/152776686-3b662e85-70ed-4665-94bf-a6819ce68493.png)



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new campaign
3. Launch campaign builder
4. Add contact segment
5. Add send email action
6. Email buttons have the btn-default class
![image](https://user-images.githubusercontent.com/96085911/152776357-1d6d6a17-d694-430c-8736-2ed58a31d9c4.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10837"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
